devtools/debug.py

```diff
@@ -141,38 +194,24 @@ def breakpoint(self) -> None:
     def timer(self, name: 'Optional[str]' = None, *, verbose: bool = True, file: 'Any' = None, dp: int = 3) -> Timer:
         return Timer(name=name, verbose=verbose, file=file, dp=dp)
 
-    def _process(self, args: 'Any', kwargs: 'Any', frame_depth: int) -> DebugOutput:
+    def _process(self, args: 'Any', kwargs: 'Any', frame_depth: int, trace: bool) -> DebugOutput:
         """
         BEWARE: this must be called from a function exactly `frame_depth` levels below the top of the stack.
         """
         # HELP: any errors other than ValueError from _getframe? If so please submit an issue
         try:
             call_frame: 'FrameType' = sys._getframe(frame_depth)
         except ValueError:
-            # "If [ValueError] is deeper than the call stack, ValueError is raised"
+            # "If [the given frame depth] is deeper than the call stack,
```

Comment:
undo this change.

Reply:
Happy to undo this, but this comment made no sense to me.  I had to go find that quote in the actual python docs to figure out what it meant.